### ISSUE TITLE
Describe issue labels in the CONTRIBUTING.org

### ISSUE THREAD

**Issue by JAremko** [syl20bnr/spacemacs#5254]
I think it will be helpful for those who want to contribute if [CONTRIBUTING.org](https://github.com/syl20bnr/spacemacs/blob/master/CONTRIBUTING.org)  would have a section devoted to issue labels. 
 It probably should mention that if you are a beginner and you want to start contributing - you might want to search for open issues with `Beginner friendly` label.   Also it will help with ambiguity.  For example, is `Beginner friendly` stands for "easy issue" or for making Spacemacs more "Beginner friendly" ? Some people might be confused.

**Comment by a13ph** [comment]
Maybe rewording problematic labels would be better, if there are any, actually.

Seeing label contents clarified confusing meanings rather quickly for me

**Comment by a13ph** [comment]
Okay, my own issue :) with labels is that it's unclear on some differences
- Can issue be confirmed or needing repro steps, but still in forum and not on bugtrack? 
- What differentiates ML and forum?

We need use to describe boundaries of some labels

**Comment by StreakyCobra** [comment]
We use Github issues for 3 different purposes:

- Bug tracker (Spacemacs' bugs)
- Forum (Spacemacs quesiton/enhencement/problems)
- Mailing lists (developer discussion)

These high-level labels are not forcing anything, they are just informative. If an issue is reporting a **bug** within Spacemacs and is using the template, it goes in the "Bug tracker" category. If it is a question, an unidentified problem, an idea, a proposition of enhancement, or anything else, it goes in the "Forum". Finally issue that are clearly discussion between contributors about features of Spacemacs, or the ones which derive to a discussion, goes in the "Mailing list" category.

Boundaries are not often very clear, like a forum question that may derive to a discussion or another one that may derive to a bug report. We try to keep labels synchronized. Anyway people shouldn't care about which one of these label is used, we use them for keeping stuff organized. Not being in the "bug tracker" category doesn't means it is not a bug.

Concerning the first question, yes, there may be forum labels having the "need repro steps".

**Comment by d12frosted** [comment]
This description might find a proper place in wiki.

**Comment by AlejandroCatalina** [comment]
I was gonna say the same, it is very useful to get people to understand this. And if people do understand it we might all make a better use of this.

**Comment by StreakyCobra** [comment]
It's already [WIP](https://github.com/syl20bnr/spacemacs/wiki/Project-management).

**Comment by d12frosted** [comment]
Thanks for the link :) Didn't see it before.

**Comment by StreakyCobra** [comment]
Because it's WIP :joy:

**Comment by d12frosted** [comment]
```
$ git clone https://github.com/syl20bnr/spacemacs.wiki.git
$ cd spacemacs.wiki
$ git checkout -b develop
# make some edits
# commit them
$ git push origin develop
```

This is WIP, but you're on master! :-P

**Comment by JAremko** [comment]
Garbagecollected
